pkgs/top-level/python-packages.nix

```diff
@@ -8242,6 +8242,25 @@ in {
     };
   };
 
+  lit = buildPythonApplication rec {
+    version = "0.5";
+    name = "lit-${version}";
+
+    disabled = isPy3k;
+
+    src = pkgs.fetchurl {
+      url = "https://files.pythonhosted.org/packages/5b/a0/dbed2c8dfb220eb9a5a893257223cd0ff791c0fbc34ce2f1a957fa4b6c6f/lit-0.5.0.tar.gz";
```

Comment:
I'm not sure how to make fetchPypi work here--all the variants AFAICT give a 404 :/. I'm pushing what I have to demonstrate.  Not sure why this package would be different than others?